cciskel-duffy

```diff
@@ -21,7 +22,7 @@ import argparse
 
 url_base="http://admin.ci.centos.org:8080"
 api_key=open(os.path.expanduser('~/duffy.key')).read().strip()
-cached_state_path = os.path.expanduser('~/duffy-state-cache.json')
+cached_state_path = os.path.expanduser('~/.duffy-state-cache.json')
```

Comment:
Maybe. Mainly, my processing was around the topology -> inventory tooling. Where the duffy-state-cache.json file lives isn't of great importance to me.